kyuubi-common/src/main/scala/org/apache/kyuubi/operation/log/OperationLog.scala

```diff
@@ -262,6 +262,7 @@ class OperationLog(path: Path) {
     try {
       f
     } catch {
+      case _: NoSuchFileException =>
```

Comment:
Emm... how about logging a simple one-line message without stacktrace?

Reply:
NP, I just see the follow message about the deadlock issue.

Reply:
Yes, that bothers me a bit too.